sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSiteScriptPackage.md

```diff
@@ -0,0 +1,134 @@
+---
+external help file: Microsoft.Online.SharePoint.PowerShell.dll-Help.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version:
+schema: 2.0.0
+---
+
+# Set-SPOSiteScriptPackage
+
+## SYNOPSIS
+Updates a previously uploaded site script package. The package file must be a zip archive containing all the files necessary for the site script. A file called “manifest.json” with script actions must be present in this zip file.
+
+## SYNTAX
+
+```
+Set-SPOSiteScriptPackage -Identity <SPOSiteScriptPipeBind> [-Title <String>] [-ContentPath <String>]
+ [-Description <String>] [-Version <Int32>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Updates a previously uploaded site script package. The package file must be a zip archive containing all the files necessary for the site script. A file called “manifest.json” with script actions must be present in this zip file.
+
+## EXAMPLES
+
+### Example
+This example updates a site script package as a zip file containing a manifest.json script and an updated Dataverse solution zip file. The content of manifest.json is as follows:
```

Comment:
Please move the description at the end of the example

Reply:
@get-itips Done, thanks.